Dockerfile

```diff
@@ -1,5 +1,10 @@
 FROM python:3.6.5-slim-stretch
```

Comment:
I was really hoping to be able to do ```FROM joplin-base``` here but I couldn't figure out how to get Heroku to build the base dockerfile into an image before trying to build this one, so for now there's a bit of copypasta.

I'd like to put this off into an issue and tag it as techdebt if y'all are cool with that instead of doing more digging right now.

Reply:
+1 to bookmarking it as devops tech debt in a gh-issue

Reply:
https://github.com/cityofaustin/techstack/issues/604